Changes to do with select

inaimathi opened the pull request:
- Add select box option to tabs menu
- Factor out tabs into components module
- Add vizlite.js as a dep

inaimathi committed:
Changes to do with select

- Add select box option to tabs menu
- Factor out tabs into components module
- Add vizlite.js as a dep

jamesaoverton commented:
This is working for me in Firefox.

In Safari and Chrome I can see the select bar and select an editor, but then I still see just the Turtle editor. No helpful messages in the browser consoles.

inaimathi committed:
Cross-browser select replaces tabs

- use jQuery to set up tab interaction in the `select` element
  case (confirmed working in Firefox and Chrome)

inaimathi committed:
Only do the select hack in non-FF browsers

jamesaoverton commented:
Working for me in FF, Safari, Chrome. Please merge.
